hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/meta/CkpMetadata.java

```diff
@@ -175,8 +175,8 @@ public String lastPendingInstant() {
     load();
     if (this.messages.size() > 0) {
       CkpMessage ckpMsg = this.messages.get(this.messages.size() - 1);
-      // consider 'aborted' as pending too to reuse the instant
-      if (!ckpMsg.isComplete()) {
+      // should not consider 'aborted', which will lead writer instant not consistent with coordinator
+      if (ckpMsg.isInflight()) {
         return ckpMsg.getInstant();
```

Comment:
We never abort a checkpoint in master code now, can you double check this logic again ?

Reply:
Thanks, no abort in coordinator, I will check other situation of abort instant.